src/java.base/share/classes/jdk/internal/util/DecimalDigits.java

```diff
@@ -132,6 +134,53 @@ public static int stringSize(long x) {
         return 19 + d;
     }
 
+    /**
+     * Determine whether the two strings in bytes are both numbers. If they are, return d0 * 10 + d1, otherwise return -1
```

Comment:
This is confusing: two strings?
Also, "digits" would be better than "numbers".
Finally, d0 and d1 are not defined here.